src/Particles.jl

```diff
@@ -59,6 +61,8 @@ struct ParticleInfo
     latex::String
 end
 
+Particle(pdgid) = _current_particle_dct[pdgid]
```

Comment:
PDG-ID is not unique ... I guess it differs for example from the numbering scheme of GEANT4

Reply:
I know, we need to catch such errors